### DISCUSSION TITLE
Keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by katarinaslama** [microsoft/vscode-jupyter#9593] [Questions and Answers] [answered]
In addition to the keyboard shortcut for `run cell`, are there also keyboard shortcuts for `run above` and `debug cell`? Or is there a straightforward way to create those shortcuts? Thanks!

**Comment by DonJayamanne** [comment, marked as answer]
Yes this is possible
* Go into the command palette and select the command `Preferences: Open Keyboard Shortcuts`
* Search for the desired command and add a custom keyboard shortcut
See here for more details on this https://code.visualstudio.com/docs/getstarted/keybindings

![image](https://user-images.githubusercontent.com/1948812/161836102-f42f653f-31d4-4fce-80cd-1f8647d20fd9.png)
